README.md

```diff
@@ -1,6 +1,10 @@
 # algorand-sdk-testing
+
 Testing files for Algorand SDKs
```

Comment:
[Markdownlint](https://github.com/DavidAnson/markdownlint/tree/v0.24.0) which I have as a VS-code extension, re-spaced the README. Are we okay with these changes?

Reply:
Looks good to me. If you want to avoid disabling rules you could fix the headers too.

Reply:
👍